js/src/format/Schema_generated.ts

```diff
@@ -0,0 +1,2254 @@
+/* tslint:disable:class-name */
+// automatically generated by the FlatBuffers compiler, do not modify
+
+import { flatbuffers } from 'flatbuffers';
```

Comment:
Can any of these files be automatically generated as part of the build? I know some of the others have manual modifications

Reply:
@wesm unfortunately no ([details here](https://github.com/graphistry/arrow-1/blob/5f66c097d31e2d3bf3ddefb249d697fb61b23ebd/js/DEVELOP.md#updating-the-arrow-format-flatbuffers-generated-code)).

The `/* tslint:disable:class-name */` is a directive to ignore the [tslint "class-name"](https://palantir.github.io/tslint/rules/class-name/) rule that enforces PascalCased class names. The offending classname here is the type `Struct_` defined in Schema.fbs. Alternatively, we can turn this rule off by setting [this property](https://github.com/graphistry/arrow-1/blob/5f66c097d31e2d3bf3ddefb249d697fb61b23ebd/js/tslint.json#L6) to `false`.

Since we don't have Tensors right now, we delete the `Tensor_generated.js` and `ts` files (otherwise they'll be included in the compiled source and never used, contributing unnecessary size to the JS over the wire).

When we do though, the `Tensor_generated.ts` file has the following wrong type signature for the `Tensor.addType` method, so will still require manual fixing. Alternatively getting a fix into the flatbuffers code generator would also work:

```ts
static addType(builder:flatbuffers.Builder, typeOffset: NS16187549871986683199.flatbuffers.Offset) {
  builder.addFieldOffset(1, typeOffset, 0);
};
```

Should be:
```ts
static addType(builder:flatbuffers.Builder, typeOffset: flatbuffers.Offset) {
  builder.addFieldOffset(1, typeOffset, 0);
};
```

Reply:
I wrote up an issue for the `Tensor_generated.ts` problem when I was trying to start using the TS generated files: https://github.com/google/flatbuffers/issues/4365